apis/python/src/tiledbsoma/__init__.py

```diff
@@ -156,7 +156,7 @@
     get_SOMA_version,
     get_storage_engine,
     show_package_versions,
-    verify_core_versions,
+    _verify_core_versions,
```

Comment:
Sorry, I wasn't clear -- let's keep this `tiledbsoma._general_utilities.verify_core_versions` but just don't auto-export it in the package-level `__init__.py`

Reply:
@ryan-williams so sorry!! I misread this as you populating `__all__` -- but you very much are not

Let's just call it `verify_core_versions` and this will be fine